.travis.yml

```diff
@@ -239,10 +241,44 @@ jobs:
             - libopenmpi-dev
             - openmpi-bin
       script: *script-cpp-unit
+    # Clang 7.0.0 + Python 3.7.1 @ Xenial
+    - <<: *test-cpp-unit
+      name: clang@7.0.0 +MPI +PY@3.7 +H5 +ADIOS1
+      dist: xenial
+      language: python
+      python: "3.7"
+      env:
+        - CXXSPEC="%clang@7.0.0" USE_MPI=ON USE_PYTHON=ON USE_HDF5=ON USE_ADIOS1=ON USE_ADIOS2=OFF USE_SAMPLES=ON
+      compiler: clang
+      addons:
+        apt:
+          <<: *apt_common_sources
+          packages:
+            - *clang50_deps
+            - libopenmpi-dev
+            - openmpi-bin
+      script: *script-cpp-unit
+      before_install: &clang_init
+        - CXX=clang++ && CC=clang && CXXFLAGS="-Werror -Wno-deprecated"
```

Comment:
`-Wno-deprecated` is needed since pybind11 is not yet included as `SYSTEM` and uses a deprecated Python API in 3.7+.
Both include and issue are fixed in upcoming releases of pybind11 (v2.3.0).